code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -378,11 +378,6 @@
 	weapon_type = /obj/item/weapon/gun/rifle/m41a/training
 	ammo_type = /obj/item/ammo_magazine/rifle/rubber
 
-/obj/structure/closet/crate/weapon/training/l42a
```

Comment:
should probably add training ammo for the m4ra too

Reply:
good point - would need sprites.